arch/arm/core/cortex_m/Kconfig

```diff
@@ -176,8 +211,8 @@ config XIP
 	default y
 endmenu
 
-menu "ARM Cortex-M0/M0+/M3/M4/M7 options"
-	depends on ARMV6_M || ARMV7_M
+menu "ARM Cortex-M0/M0+/M3/M4/M7/M23/M33 options"
```

Comment:
Nit: the M23/M33 should be in the second patch.

Reply:
Sure